PendingReleaseNotes

```diff
@@ -57,6 +57,12 @@
   level storage operator (like Rook) to recreate the missing file system.
   See https://docs.ceph.com/en/latest/cephfs/administration/#file-systems
   docs for more information.
+* RADOS: A POOL_APP_NOT_ENABLED health warning will now be reported if
+  the application is not enabled for the pool irrespective of whether
+  the pool is in use or not. Always add ``application`` label to a pool
```

Comment:
I think these are more commonly referred to as application tags, not labels.  It's `tag` in the OSD cap syntax too:
```
  There are two alternative forms of the optional ``{match-spec}`` syntax::

        pool={pool-name} [namespace={namespace-name}] [object_prefix {prefix}]

        [namespace={namespace-name}] tag {application} {key}={value}
```
```suggestion
  the pool is in use or not. Always tag a pool with an application using ``ceph osd pool application enable`` command
```